pkg/cmd/driftctl_test.go

```diff
@@ -58,6 +58,23 @@ func TestDriftctlCmd_Version(t *testing.T) {
 	}
 }
 
+func TestDriftctlCmd_Completion(t *testing.T) {
```

Comment:
This test is not overlapping `TestCompletionCmd` ?

Reply:
Yes in its purpose but we initialize driftctl command first instead of a random root command for the others

Reply:
I see 👍🏻